File: app/src/main/java/com/semisonfire/cloudgallery/ui/main/settings/SettingsFragment.java
package com.semisonfire.cloudgallery.ui.main.settings;

import android.os.Bundle;
import android.support.annotation.NonNull;
import android.support.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.semisonfire.cloudgallery.R;
import com.semisonfire.cloudgallery.ui.base.BaseFragment;

public class SettingsFragment extends BaseFragment {

    @Nullable
    @Override
    public View onCreateView(
            @NonNull LayoutInflater inflater,
            @Nullable ViewGroup container,
            @Nullable Bundle savedInstanceState
    ) {
        super.onCreateView(inflater, container, savedInstanceState);
        return inflater.inflate(R.layout.fragment_settings, container, false);
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);
        bind();
    }

    @Override
    public void bind() {
        getFloatButton().hide();
        getSwipeRefreshLayout().setEnabled(false);
    }

//    @Override
//    public void onTokenLoaded(String token) {
//        /* Not needed */
//    }

    @Override
    public void onInternetUnavailable() {
        /* Not require because data is not requested */
    }
}

File: app/src/main/java/com/semisonfire/cloudgallery/ui/settings/SettingsFragment.kt
package com.semisonfire.cloudgallery.ui.settings

import android.view.View
import androidx.swiperefreshlayout.widget.SwipeRefreshLayout
import com.google.android.material.floatingactionbutton.FloatingActionButton
import com.semisonfire.cloudgallery.R
import com.semisonfire.cloudgallery.core.mvp.MvpView
import com.semisonfire.cloudgallery.core.ui.BaseFragment
import com.semisonfire.cloudgallery.ui.settings.model.SettingsViewModel

interface SettingsView : MvpView<SettingsViewModel>

class SettingsFragment : BaseFragment<SettingsViewModel, SettingsView, SettingsPresenter>() {

    override fun bind(view: View) {
        super.bind(view)
        activity?.let {
            it.findViewById<FloatingActionButton>(R.id.btn_add_new).hide()
            it.findViewById<SwipeRefreshLayout>(R.id.swipe_refresh).isEnabled = false
        }
    }

    override fun layout(): Int {
        return R.layout.fragment_settings
    }
}